tests/TestBuildVolume.py

```diff
@@ -163,6 +163,7 @@ def test_computeDisalowedAreasMutliExtruder(self, build_volume):
 
 class TestUpdateRaftThickness:
     setting_property_dict = {"raft_base_thickness": {"value": 1},
+                             "raft_interface_layers": {"value": 2},
                              "raft_interface_thickness": {"value": 1},
                              "raft_surface_layers": {"value": 1},
```

Comment:
```suggestion
                             "raft_surface_layers": {"value": 3},
```

Would be nice to throw a test for multiple surface layers in too. Made it 3 instead of 2 so it is clearer which is the issue when the test fails.